core/controllers/base.py

```diff
@@ -119,6 +119,9 @@ class PageNotFoundException(Exception):
     class InternalErrorException(Exception):
         """Error class for an internal server side error (error code 500)."""
 
+    class TemporaryMaintenanceException(Exception):
+        """Error class for a temporary maintenance (error code 503)."""
```

Comment:
Let's explain this a bit more:

Error class for when the server is currently down for temporary maintenance (error code 503).